app/src/main/java/br/org/cesar/discordtime/stickysessions/data/remote/interceptor/InternetNeeded.java

```diff
@@ -0,0 +1,7 @@
+package br.org.cesar.discordtime.stickysessions.data.remote.interceptor;
+
+public interface InternetNeeded {
```

Comment:
I didn't find anyone expecting an InternetNeeded instance. Also didn't quite get it the meaning of interface name. Are you trying to say the following? "hey, if this kind of interceptor requires internet, please implement the isConnect() method." If so, someone outside class would need to control isConnect calling, don't you think?